main.go

```diff
@@ -37,6 +41,11 @@ func main() {
 	}
 
 	setupFlags(cmdServe)
+	parseFlags(cmdServe, os.Args[1:])
+	if err := loadDaemonFile(&cfg, cmdServe.Flags()); err != nil {
+		logrus.Errorf("failed to load daemon file: %s", err)
+		os.Exit(2)
```

Comment:
Exit code 2 means Misuse of shell builtins (according to Bash documentation) according to https://shapeshed.com/unix-exit-codes/.

Is it reasonable?

Reply:
is this make sense `128 - Invalid argument to exit` ?

Reply:
I think exit code 1 is OK, since it is a general error. 😄

Reply:
updated.